packages/lesmis/src/api/queries/useEmailVerification.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import { useQuery } from 'react-query';
+import { useUrlSearchParams } from '../../utils/useUrlSearchParams';
+import { get } from '../api';
+
+export const useEmailVerification = () => {
+  const [{ verifyEmailToken }] = useUrlSearchParams();
+
+  return useQuery(['user', verifyEmailToken], () => get(`verify/${verifyEmailToken}`), {
```

Comment:
Does `web-frontend` use a `get` as well? I would have thought this would be a `post`

Reply:
Yes web frontend is a get too